index.js

```diff
@@ -45,20 +49,36 @@ var Input = React.createClass({
 
     var classes = this.getInitialClasses();
 
+    var inputProps = {};
+    if (typeof this.props.value === 'string') {
+      // make input controlled
+      inputProps['value'] = this.props.value;
```

Comment:
I think we should make it a default. Simply add `value={this.props.value}` to `<input>`.